sensing/pointcloud_preprocessor/src/concatenate_data/concatenate_data_nodelet.cpp

```diff
@@ -323,22 +351,20 @@ void PointCloudConcatenateDataSynchronizerComponent::setPeriod(const int64_t new
 }
 
 void PointCloudConcatenateDataSynchronizerComponent::cloud_callback(
-  const sensor_msgs::msg::PointCloud2::ConstSharedPtr & input_ptr, const std::string & topic_name)
+  const sensor_msgs::msg::PointCloud2::SharedPtr & input_ptr, const std::string & topic_name)
```

Comment:
I think this diff is **NOT** necessary :thinking: 

FYI
https://github.com
![Screenshot from 2022-02-03 09-53-16](https://user-images.githubusercontent.com/44889564/152263230-1fcacae6-4e9b-4418-a910-1ed7f6c8c5b5.png)
/tier4/AutowareArchitectureProposal.iv/pull/792/files

Reply:
You're not using `reinterpret_cast`, are you?

Reply:
I can not use `PointCloud2Iterator` with `sensor_msgs::msg::PointCloud2::ConstSharedPtr`.
Do you have any solution?

Reply:
OK, I see. I understand that It is necessary for `PointCloud2Iterator` :+1: